components/pedigree/resources/src/main/resources/pedigree/view/exportSelector.js

```diff
@@ -185,7 +185,7 @@ define([
                         on0 : function(response) {
                             response.request.options.onFailure(response);
                         },
-                        parameters: {"image": exportString}
+                        parameters: {"image": exportString, "filename": patientDocument + ".png"}
```

Comment:
I wonder if it would be a good idea to also include a timestamp in the name; not just for the image, but for all export formats.